tip-wlan/charts/wlan-prov-service/templates/configmap.yaml

```diff
@@ -4,4 +4,16 @@ metadata:
   name: {{ include "common.fullname" . }}-log-config
   namespace: {{ include "common.namespace" . }}
 data:
+  datasource.properties: |-
+    singleDataSource.url=jdbc:postgresql://tip-wlan-postgresql:5432/prov_db
+    singleDataSource.username={{ .Values.creds.postgres.singleDataSourceUsername }}
+    singleDataSource.password={{ .Values.creds.postgres.singleDataSourcePassword }}
+    singleDataSource.driverClass=org.postgresql.Driver
+    singleDataSource.ssl=true
+    singleDataSource.sslmode=verify-ca
+    singleDataSource.sslcert=/opt/tip-wlan/certs/postgresclientcert.pem
+    singleDataSource.sslfactory=org.postgresql.ssl.LibPQFactory
+    singleDataSource.sslkey=/opt/tip-wlan/certs/postgresclient.p12
+    singleDataSource.sslrootcert=/opt/tip-wlan/certs/cacert.pem
+    singleDataSource.sslkeypassword=mypassword
```

Comment:
Please revert this to DUMMY_PASSWORD, (until we change this file into a ConfigMap/Secret and read it from override file)